.travis.yml

```diff
@@ -4,6 +4,7 @@ cache:
 
 before_cache:
   - rm -rf $HOME/.m2/repository/org/gradoop/
+  - rm -rf /tmp/_accumulo_*
```

Comment:
I don't think you need to clean up temp folders on the CI machine. Or am I wrong?